src/lib.rs

```diff
@@ -685,6 +685,8 @@ pub fn build_with_subcommand(parameters: CompileParameters) -> Result<(), Diagno
 
             let target = get_target_triple(compile_options.target.as_deref());
 
+            let error_format = project.error_format.unwrap_or_else(|| ErrorFormat::Clang);
```

Comment:
Default format is rich, we also have a way to find it in the build with params, or it might be that if it remains None it will automatically be rich